tests/integration/linkaccount_test.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2023 The Forgejo Authors. All rights reserved.
```

Comment:
This seemed to have not been caught by CI, could you update it per the convo in other PR?